### DISCUSSION TITLE
post/seven-months-of-struggle-main-part-1

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#15] [General]
# post/seven-months-of-struggle-main-part-1

长话短说，我其实已经推倒重写好几版了 由于时间跨度有些大，没有当时的感觉真的写不出有当时的味道的文字 而我现在必须挑重点把这些事情记录下来，质量不能作保证，毕竟真的不能再拖了 诸君将就着看吧👀  八月｜简历与处女面   时间回到 2023 年 8 月 22 日，晚上 8 点 13 分 浙江省杭州市

https://nickxu.me/post/seven-months-of-struggle-main-part-1

**Comment by jaketmoon** [comment]
💪

**Comment by ikun09** [comment]
加油，哥们

**Comment by dasi0227** [comment]
不知不觉看完了，目前刚大三，能初步感受到你的压力，一起加油！
